### PR TITLE
Update Docs Card Component Styling

### PR DESCRIPTION
**Pull request by connorwinston**
Improved the affordance of the docs card component link.
2023-05-15 at 15 09 31 - Silver Tapir

### CODE REVIEW

**Review by rijkvanzanten — APPROVED** [review]
Code LGTM!

**Comment by rijkvanzanten** [comment]
Cc @benhaynes for a quick design review

**Review by benhaynes — APPROVED** [review]
This looks awesome!!

**Review by benhaynes — APPROVED** [review]
This looks awesome!!

**Review by phazonoverload — APPROVED** [review]
Lovely, thanks for doing this.

File: docs/.vitepress/components/Card.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@ const props = defineProps({
 	title: { type: String, required: true },
 	h: { type: String, required: false, default: '2' },
 	text: { type: String, required: true },
-	icon: { type: String, required: false, default: null },
+	icon: { type: String, required: false, default: "/icons/card_link.svg" },
```
Comment (paescuj) [inline docs/.vitepress/components/Card.vue:6]:
```suggestion
	icon: { type: String, required: false, default: '/icons/card_link.svg' },
```

File: .changeset/strange-ears-pull.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"docs": patch
+---
+
+Improved Affordance of Docs Card Component Link
```
Comment (paescuj) [inline .changeset/strange-ears-pull.md:5]:
```suggestion
Improved affordance of Docs Card Component Link
```

**Review by paescuj — APPROVED** [review]

**Pull request merged**